template/src/Autovis/printTime.js

```diff
@@ -0,0 +1,7 @@
+import pad from './pad';
+
+export default printTime = (currentTime) => {
```

Comment:
no need to give this a name, just use `export default (currentTime) => {`

Reply:
done